azerothcore/azerothcore-wotlk
# fix(DB/quest_poi_points): location tracker of Quell the Uprising

## Changes Proposed:
-  fix location tracker (quest poi point in the map)

## Issues Addressed:
- Closes https://github.com/chromiecraft/chromiecraft/issues/408
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/5264


## Tests Performed:
- tested in-game


## How to Test the Changes:
.q add 387
.tele stormwind
open the map and see if the quest tracker location is on the The Stockade entrance


## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
